documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

```diff
@@ -56,23 +58,27 @@ external:
     validIssuerUri: <__https://<auth-server-address>/auth/realms/external__> <2>
     jwksEndpointUri: <__https://<auth-server-address>/auth/realms/external/protocol/openid-connect/certs__> <3>
     userNameClaim: preferred_username <4>
-    tlsTrustedCertificates: <5>
+    maxSecondsWithoutReauthentication: 3600 <5>
+    tlsTrustedCertificates: <6>
     - secretName: oauth-server-cert
       certificate: ca.crt
-    disableTlsHostnameVerification: true <6>
-    jwksExpirySeconds: 360 <7>
-    jwksRefreshSeconds: 300 <8>
-    enableECDSA: "true" <9>
+    disableTlsHostnameVerification: true <7>
+    jwksExpirySeconds: 360 <8>
+    jwksRefreshSeconds: 300 <9>
+    jwksMinRefreshPauseSeconds: 1 <10>
+    enableECDSA: "true" <11>
 ----
 <1> Listener type set to `oauth`.
 <2> URI of the token issuer used for authentication.
 <3> URI of the JWKS certificate endpoint used for local JWT validation.
 <4> The token claim (or key) that contains the actual user name in the token. The user name is the _principal_ used to identify the user. The `userNameClaim` value will depend on the authentication flow and the authorization server used.
-<5> (Optional) Trusted certificates for TLS connection to the authorization server.
-<6> (Optional) Disable TLS hostname verification. Default is `false`.
-<7> The duration the JWKs certificates are considered valid before they expire. Default is `360` seconds. If you specify a longer time, consider the risk of allowing access to revoked certificates.
-<8> The period between refreshes of JWKs certificates. The interval must be at least 60 seconds shorter than the expiry interval. Default is `300` seconds.
-<9> (Optional) If ECDSA is used for signing JWT tokens on authorization server, then this needs to be enabled. It installs additional crypto providers using BouncyCastle crypto library. Default is `false`.
+<5> (Optional) Activates the Kafka re-authentication mechanism that enforces session expiry to the same length of time as the access token. If the specified value is less than the time left for the access token to expire, then the client will have to re-authenticate before the actual token expiry. By default, the session does not expire when the access token expires, and the client does not attempt re-authentication.
```

Comment:
@mstruk - How many seconds should the user set for `maxSecondsWithoutReauthentication`? I think we should provide some guidance here?

Reply:
In the example it is set to 3600 seconds (1 hour) which is our guidance.